api/src/main/java/org/apache/gravitino/stats/PartitionRange.java

```diff
@@ -38,6 +38,9 @@ public class PartitionRange {
 
   private PartitionRange() {}
 
+  /** A PartitionRange that includes all partitions. */
+  public static final PartitionRange ALL_PARTITIONS = new PartitionRange();
```

Comment:
is it nesseary to update related document?

Reply:
We don't have similar documents.